tutorials/auto_scheduler/tune_conv2d_layer_cuda.py

```diff
@@ -82,17 +84,19 @@ def conv2d_layer(N, H, W, CO, CI, KH, KW, stride, padding):
 # * :code:`num_measure_trials` is the number of measurement trials we can use during the search.
 #   We only make 10 trials in this tutorial for a fast demonstration. In practice, 1000 is a
 #   good value for the search to converge. You can do more trials according to your time budget.
-# * In addition, we use :code:`RecordToFile` to dump measurement records into a file `conv2d.json`.
+# * In addition, we use :code:`RecordToFile` to dump measurement records into a file.
+#   Note that here we use a temporarty file for demonstraction, but in practice you should use
+#   a more maintainable file name such as `conv2d.json`.
 #   The measurement records can be used to query the history best, resume the search,
 #   and do more analyses later.
 # * see :any:`auto_scheduler.TuningOptions`,
 #   :any:`auto_scheduler.LocalRPCMeasureContext` for more parameters.
-
+logfile = tempfile.NamedTemporaryFile(prefix="conv2d", suffix=".json")
```

Comment:
can we use tvm.contrib.util.tempdir instead? It will be able to clean up the content at exit

Reply:
Sure. In that case do we still need to change `task_python_docs.sh`?

Reply:
Another concern is that since this is a tutorial, it would be a bit weird if users run this tutorial locally but the generated log was removed.

Reply:
The updated changes now explicitly uses a directory to keep tuning logs, and we deleted it in the testing script.

@tqchen @merrymercy please check if this solution works for you.